source/core/slang-nvrtc-compiler.cpp

```diff
@@ -307,6 +307,9 @@ SlangResult NVRTCDownstreamCompiler::compile(const CompileOptions& options, RefP
         // This is arguably too much - but nvrtc does not appear to have a mechanism to switch off individual warnings.
         // I tried the -Xcudafe mechanism but that does not appear to work for nvrtc
         cmdLine.addArg("-w");
+
+        //
+        cmdLine.addArg("-arch=compute_70");
```

Comment:
We should eventually add a new "profile family" to capture the CUDA target capabilities that are based on architecture version. That would require adding logic to mark certain builtins as requiring a specific version/architecture, similar to how we already do things for GLSL.

Whether to do this soon, or wait for a more complete/holistic approach to handling profiles/capabilities is unclear.